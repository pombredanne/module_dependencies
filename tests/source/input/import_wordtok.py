from nltk import word_tokenize

tokenizer = word_tokenize
