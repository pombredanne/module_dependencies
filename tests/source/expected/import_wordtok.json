{
    "dependencies": ["nltk.word_tokenize"],
    "imports": ["nltk"]
}
